# 
# Submitted-By:
# Committed-By: Bjørn Bürger, 2006-01-03
#
# Error: 
# 
# not gcc4 save, -mcpu is deprecated
#
# Description: 
#
# -march may be used alternatively
#
# State:
# 
# <upstream state of this problem>
#

diff -urN mknbi-1.4.4.orig/Makefile mknbi-1.4.4/Makefile
--- mknbi-1.4.4.orig/Makefile	2006-01-03 13:58:14.000000000 +0100
+++ mknbi-1.4.4/Makefile	2006-01-03 14:01:20.000000000 +0100
@@ -18,7 +18,6 @@
 FIRSTRELOCS=	0x92800 0x82800
 OLDGAS:=	$(shell $(AS) --version | grep -q '2\.9\.1' && echo -DGAS291)
 CFLAGS=		-I. -Os -ffreestanding -fstrength-reduce -fomit-frame-pointer \
-		-mcpu=i386 \
 		-Wall -W -Wno-format -Wno-unused -DVERSION=\"$(VERSION)$(EXTRAVERSION)\"
 LDBINARYFLAG=	--oformat binary
 FIRST32SIZE=	6144
diff -urN mknbi-1.4.4.orig/nfl.c mknbi-1.4.4/nfl.c
--- mknbi-1.4.4.orig/nfl.c	2003-09-22 16:47:08.000000000 +0200
+++ mknbi-1.4.4/nfl.c	2006-01-03 13:59:22.000000000 +0100
@@ -124,7 +124,7 @@
 static unsigned short menu_cols = 0x0800;
 
 /* Terminal types.  */
-static int terminal = TERMINAL_CONSOLE;
+int terminal = TERMINAL_CONSOLE;
 
 /* Defined in ANSI.C */
 extern unsigned short rows, columns, attr;
